skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -176,3 +184,48 @@ val SkikoPointerEvent.isRightClick: Boolean
 val SkikoPointerEvent.isMiddleClick: Boolean
     get() = button.has(SkikoMouseButtons.MIDDLE) && (kind == SkikoPointerEventKind.UP)
 
+/**
+ * The device type that produces pointer events, such as a mouse or stylus.
+ */
+enum class SkikoPointerDevice {
+    MOUSE, TOUCH, UNKNOWN
+}
+
+/**
+ * Represents pointer such as mouse cursor, or touch/stylus press.
+ * There can be multiple pointers on the screen at the same time.
+ */
+data class SkikoPointer(
+    /**
+     * X position in points (scaled pixels that depend on the scale factor of the current display)
+     */
+    val x: Double,
+    /**
+     * Y position in points (scaled pixels that depend on the scale factor of the current display)
+     */
+    val y: Double,
+
+    /**
+     * `true` if the pointer event is considered "pressed." For example, finger
+     *  touching the screen or a mouse button is pressed [pressed] would be `true`.
+     *  During the up event, pointer is considered not pressed.
+     */
+    val pressed: Boolean,
+
+    /**
+     * The device type associated with the pointer, such as [mouse][SkikoPointerDevice.MOUSE],
+     * or [touch][SkikoPointerDevice.TOUCH].
+     */
+    val device: SkikoPointerDevice = SkikoPointerDevice.MOUSE,
+
+    /**
+     * Unique id associated with the pointer. Used to distinguish between multiple pointers that can exist
+     * at the same time (i.e. multiple pressed touches).
+     */
+    val id: Long = 0,
```

Comment:
Done.

Was not sure about this one, because in some cases we need a simple API for sending only one pointer.

But after a second thought, I realized that it can be solved by additional helper methods. For example, instead of  `sendEvent(listOf(Pointer(3, 5)))`, we can add `sendEvent(3, 5)`